spec/services/swiss_tables_spec.rb

```diff
@@ -1,27 +1,24 @@
 # frozen_string_literal: true
 
 RSpec.describe SwissTables do
-  let(:alice) { create(:player, name: 'Alice') }
-  let(:bob) { create(:player, name: 'Bob') }
-  let(:charlie) { create(:player, name: 'Charlie') }
-  let(:dave) { create(:player, name: 'Dave') }
-  let(:eddie) { create(:player, name: 'Eddie') }
-  let(:florence) { create(:player, name: 'Florence') }
-
-  let(:alice_bob) { create(:pairing, player1: alice, player2: bob) }
-  let(:charlie_dave) { create(:pairing, player1: charlie, player2: dave) }
-  let(:eddie_florence) { create(:pairing, player1: eddie, player2: florence) }
-
-  let(:alice_bye) { create(:pairing, player1: alice, player2: nil) }
-  let(:alice_florence) { create(:pairing, player1: alice, player2: florence) }
-  let(:bob_charlie) { create(:pairing, player1: bob, player2: charlie) }
-  let(:dave_eddie) { create(:pairing, player1: dave, player2: eddie) }
+  let(:alice) { PairingStrategies::PlainPlayer.new(1, 'Alice', true, false) }
+  let(:bob) { PairingStrategies::PlainPlayer.new(2, 'Bob', true, false, points: 6) }
+  let(:charlie) { PairingStrategies::PlainPlayer.new(3, 'Charlie', true, false) }
+  let(:dave) { PairingStrategies::PlainPlayer.new(4, 'Dave', true, false) }
+  let(:eddie) { PairingStrategies::PlainPlayer.new(5, 'Eddie', true, false, points: 6) }
+  let(:florence) { PairingStrategies::PlainPlayer.new(6, 'Florence', true, false, points: 6) }
+
+  let(:alice_bob) { PairingStrategies::PlainPairing.new(alice, 0, bob, 0) }
+  let(:charlie_dave) { PairingStrategies::PlainPairing.new(charlie, 0, dave, 0) }
+  let(:eddie_florence) { PairingStrategies::PlainPairing.new(eddie, 0, florence, 0) }
+
+  let(:alice_bye) { PairingStrategies::PlainPairing.new(alice, 0, nil, 0) }
+  let(:alice_florence) { PairingStrategies::PlainPairing.new(alice, 0, florence, 0) }
+  let(:bob_charlie) { PairingStrategies::PlainPairing.new(bob, 0, charlie, 0) }
+  let(:dave_eddie) { PairingStrategies::PlainPairing.new(dave, 0, eddie, 0) }
 
   describe '#assign_table_numbers!' do
     it 'sorts non-bye pairings by number of points' do
-      alice_florence.update(score1: 0, score2: 6)
-      bob_charlie.update(score1: 6, score2: 0)
-      dave_eddie.update(score1: 0, score2: 6)
```

Comment:
It might be nice to set the points explicitly here rather than needing to refer to the above data? It might be easier to read if we avoid setting points in the setup and have the tests set it directly?